accurest-core/src/test/groovy/io/codearte/accurest/dsl/WireMockGroovyDslSpec.groovy

```diff
@@ -80,23 +80,23 @@ class WireMockGroovyDslSpec extends WireMockSpec {
 		when:
 			String wireMockStub = new WireMockStubStrategy(groovyDsl).toWireMockClientStub()
 		then:
-			new JsonSlurper().parseText(wireMockStub) == new JsonSlurper().parseText('''
+		JSONAssert.assertEquals('''
 {
-    "request": {
-        "method": "GET",
-        "headers": {
-            "Content-Type": {
-                "equalTo": "application/vnd.pl.devoxx.aggregatr.v1+json"
-            }
-        },
-        "url": "/ingredients"
-    },
-    "response": {
-        "status": 200,
-        "body": "{\\"ingredients\\":[{\\"type\\":\\"MALT\\",\\"quantity\\":100},{\\"type\\":\\"WATER\\",\\"quantity\\":200},{\\"type\\":\\"HOP\\",\\"quantity\\":300},{\\"type\\":\\"YIEST\\",\\"quantity\\":400}]}"
+  "request" : {
+    "url" : "/ingredients",
+    "method" : "GET",
+    "headers" : {
+      "Content-Type" : {
+        "equalTo" : "application/vnd.pl.devoxx.aggregatr.v1+json"
+      }
     }
+  },
+  "response" : {
+    "status" : 200,
+    "body" : "{\\"ingredients\\":[{\\"type\\":\\"MALT\\",\\"quantity\\":100},{\\"type\\":\\"WATER\\",\\"quantity\\":200},{\\"type\\":\\"HOP\\",\\"quantity\\":300},{\\"type\\":\\"YIEST\\",\\"quantity\\":400}]}"
+  }
 }
-''')
+''', wireMockStub, false)
```

Comment:
Generally it's good to extract such 'false' magic to named constant describing what it does